CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+Contributing
```

Comment:
What about moving this file to the `docs` directory? Remember to update the places where it is called.

Reply:
I will do it. I believed that the 'CONTRIBUTING.md' directory should be in the root of the project, but I see here does not have this pattern.

https://help.github.com/articles/setting-guidelines-for-repository-contributors/

Reply:
Commit made because this review: 18ca085